index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
```

Comment:
Remove empty spaces from your code

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear. eg why having the empty lines in your code base a bad thing